script/Ciel/Day 3/03_01_CIEL03_3_3B.txt

```diff
@@ -76,32 +82,33 @@ Senpai's hand reaches out.
 [sha:68e7dbfa48953f2e7130a9a7bc9e1d407dad7d37]{
 -- Page 197, Offset 5353.
 -- すれ違った俺の肩を呼び止めるように。
-As if trying to stop me as I pass by.
+She goes to grab my shoulder as I pass her by.
 }
 [sha:cb613451f2a7140472b12bf407f02c96c19c5030]{
 -- Page 197, Offset 5354.
 -- それを振りほどこうとして、
-I try to shake it off―――
+When I try to shake her off―――
 }
 [sha:67d9cae6e82c27cd9eb8e0a3ed4a7ced04800a87]{
 -- Page 198, Offset 5355.
 -- 何の悪意もない偶然で、世界の在り方が一変した。
-Through mere coincidence, without any malice, the way of the world changes completely.
+Through no fault of anyone's, the world is turned on its head.
 }
 [sha:fc8c939775928b159372a3d43b15349ad9ca7e95]{
 -- Page 199, Offset 5356.
 -- 『――――――、あ』
-"――――――Ah."
+'――――――%{i}Ah%{/i}.'
 }
 [sha:4ae4840fce35af51972ed7a21c7ffa9fc6937bf5]{
 -- Page 199, Offset 5357.
 -- ズレる。ズレる。ズレる。ズレる。
-The world is contorted. Contorted. Contorted. Contorted.
+Disarray. Disorder. Dislocation. Disruption.
 }
 [sha:fcfb1f7428ec2960ab2bca7ac67804fef276deca]{
 -- Page 199, Offset 5358.
 -- 世界が断線する。一秒後の死が訪れる。
-The world is disconnected. Death is a second away.
+The world becomes disjointed.
```

Comment:
dunno about "disjointed" here. 断線する is used to describe like cable disconnections

Reply:
I picked dis_jointed_ as reference to the lines (joints). It does seem to mostly apply to wires breaking, 
> The world snaps/breaks apart.

?
My issue with disconnected is that it can read like it's disconnected as a whole _from_ something, rather than being broken apart itself

Reply:
snaps works i guess